setup.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This file is used to build a Python package that can then by used by
+Google Cloud Dataflow workers (Apache Beam).
+
+The package is build by running 'python setup.py sdist' in the build.py.
+"""
+
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import pkg_resources
+import setuptools
+
+# Configure the required packages and scripts to install.
+with open('requirements.txt', encoding='utf-8') as requirements_txt: # pylint: disable=replace-disallowed-function-calls
+    REQUIRED_PACKAGES = [
+        str(requirement)  # pylint: disable=replace-disallowed-function-calls
```

Comment:
Why do we need to put a str() around this?

Is this the correct thing to do in Py3?

Reply:
`requirements` is of `Requirement` type and it needs to be transformed to string.

Reply:
Please add a comment explaining that, it's not clear.

Reply:
Done.